code/modules/mob/living/living.dm

```diff
@@ -984,7 +1000,7 @@ Sorry Giacom. Please don't be mad :(
 			overlay_fullscreen("blind", /obj/screen/fullscreen/blind)
 	else if(eye_blind)
 		var/blind_minimum = 0
-		if(stat == UNCONSCIOUS || (disabilities & BLIND))
+		if(stat != CONSCIOUS || (disabilities & BLIND))

```

Comment:
this is so calling adjust_blindness(-X) while dead doesn't give us eye_blind = 0. We're blind as long as we're dead (even though we don't see the blindness overlay)